docs/ref/csrf.txt

```diff
@@ -136,9 +136,8 @@ and read the token from the DOM with JavaScript:
 .. code-block:: html+django
 
     {% csrf_token %}
-    <script type="text/javascript">
-    // using jQuery
-    var csrftoken = jQuery("[name=csrfmiddlewaretoken]").val();
+    <script>
+    var csrftoken = document.querySelector('[name=csrfmiddlewaretoken]').value;
     </script>
 
 Setting the token on the AJAX request
```

Comment:
Do we need to update this section too?

Reply:
I've update this section to add an example without jQuery. Let me know what you think.